src/index.css

```
@@ -246,3 +246,120 @@
+#termly-code-snippet-support .t-popup,
+#termly-code-snippet-support .t-modal,
+.termly-styles-root [class*="popup"],
+.termly-styles-root [class*="modal"],
+.termly-styles-root [class*="banner"],
+.termly-styles-root [class*="container"],
+.t-root [class*="popup"],
+.t-root [class*="modal"],
+.t-root [class*="banner"],
+.t-root [class*="container"] {
+  background-color: #111111 !important;
+  background: #111111 !important;
```

Stylelint (17.11.1)

[error] Overridden property "background-color" by shorthand "background" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)